README.md

```diff
@@ -1,4 +1,24 @@
+
+AlarmClock
+==========
+This gives the option to query whether a specified time period has expired. The utility is made to have a very low CPU clock cycle cost for the client compared to using StopWatch or std::chrono.
+
+The implementation of AlarmClock was made to be  as accurate as possible, with the assumption that low CPU usage and cheap, CPU wise,  API calls are more important than absolute accuracy in terms of some extra microseconds of expiration overhead. 
+
+Common expiration time overhead ranged from 50 - 150 microseconds on the current testing platform when the Alarm clock was checked for expiration in a continous loop. (which normally is not a good usage pattern)
```

Comment:
Continuous is misspelled in the last sentence. Also, the additional fragment in parens should come before the period in that last sentence.